pretix_eth/exporter.py

```diff
@@ -57,7 +56,7 @@ def refund_to_row(refund):
     fiat_amount = refund.amount
     token_amount = refund.info_data.get("amount", "")
 
-    wallet_address = WalletAddress.objects.filter(order_payment=refund.payment).first()
+    wallet_address = None  # todo WalletAddress.objects.filter(order_payment=refund.payment).first()
```

Comment:
out of the loop - why are we not showing wallet addresses anymore?

Reply:
We have deleted the model altogether for single receiver mode, now it's just a settings field. This was a temporary hack -- commented it out to prevent it from crashing due to the missing model. Fixed now :)

Reply:
Deleted the refund logic now, the single receiver mode doesn't support refunds for privacy reasons.